app/views/course/forum/forums/_subscribe_button.html.slim

```diff
@@ -15,21 +15,22 @@
   - subscribe_tooltip = t('course.forum.forums.subscribe.tag')
   - unsubscribe_tooltip = t('course.forum.forums.unsubscribe.tag')
 
+- disable_button = !admin_enable_email || user_unsubscribed
```

Comment:
Shouldn't we rename this to `is_disable_button` to reflect boolean state?

Reply:
It is fine as It's used locally and we are removing this rails page soon..